ci.mk

```diff
@@ -187,17 +192,16 @@ define cppcheck_misra_addon
 "{\
     \"script\": \"misra\",\
     \"args\": [\
-        \"--rule-texts=ci/misra/rules.txt\"\
+        \"--rule-texts=$(root_dir)/ci/misra/rules.txt\"\
```

Comment:
Don't really see the need for this modification. Why are you removing this?

Reply:
This will be addressed on the next fix PR.